pkg/authorization/apis/authorization/conversion.go

```diff
@@ -241,3 +243,42 @@ func Convert_rbac_PolicyRules_To_authorization_PolicyRules(in []rbac.PolicyRule)
 	}
 	return rules
 }
+
+func copyMapExcept(in map[string]string, except string) map[string]string {
+	out := map[string]string{}
+	for k, v := range in {
+		if k != except {
+			out[k] = v
+		}
+	}
+	return out
+}
+
+func convert_Origin_Annotations_To_rbac_Annotations(in map[string]string) map[string]string {
+	// can't use const in pkg/oc/admin/policy because of import cycle
+	value, ok := in["openshift.io/reconcile-protect"]
+	if ok {
+		out := copyMapExcept(in, "openshift.io/reconcile-protect")
+		if value == "true" {
+			out[rbac.AutoUpdateAnnotationKey] = "false"
+		} else {
```

Comment:
The reason why I left it there is that we have yaml files with the "other" value already (at least for rbac) and ignoring it would mean the value is lost in the conversion.
And even if it is the default, I think it is not ok to ignore an explicit value and drop it (because when that default changes, then the thing you put in there is gone, the conversion killed it).
So unless there is an actual detrimental effect in preserving explict annotations (whatever the value) I feel it is safer to keep them even if the value they represent is the default action currently taken by the bootstrap reconciliation code.